package/mines/package

```diff
@@ -1,6 +1,6 @@
 # vim: set ft=sh:
 pkgname=minesweeper
-pkgdesc="mine detection game"
+pkgdesc="Mine detection game"
 url=https://github.com/rmkit-dev/rmkit
```

Comment:
Shouldn’t this `url=` field be changed to `"https://rmkit.dev/apps/minesweeper"`?